### ISSUE TITLE
Refreshing Session Token: ERROR: Post /services/oauth2/token: unsupported protocol scheme ""

### ISSUE THREAD

**Issue by sfllaw** [ForceCLI/force#527]
If you're using a custom endpoint for tokens, and your session expires, then `refreshTokenURL` has trouble refreshing the session token:

```console
slaw@mittens:~/work/src/github.com/ForceCLI/force$ ./force fetch -t CustomObject
Refreshing Session Token
Post /services/oauth2/token: unsupported protocol scheme ""
ERROR: Post /services/oauth2/token: unsupported protocol scheme ""
```

This is because `refreshTokenURL` calls `tokenURL` while `CustomEndpoint == ''`:
https://github.com/ForceCLI/force/blob/063d47e03d5795106b67d40029b4771cd83b056c/lib/force.go#L303-L311

I think the hacky solution would be setting:
```go
CustomEndpoint = f.Credentials.InstanceUrl
```

**Comment by asknet** [comment]
I'm facing the same issue. Appreciate if this can be fixed. Thanks!

**Comment by cwarden** [comment]
Fixed in #543
